installation-linux-rpm.md

```diff
@@ -13,48 +13,21 @@ will also work on any other RPM-based distribution.
 
 ## Install RPM package from COPR repository
 
-There's a [Fedora COPR repository](https://copr.fedorainfracloud.org/coprs/leisteth/yggdrasil/) available, which provides recent versions of Yggdrasil - *not only for Fedora, but also for CentOS/RHEL!*
+There's a [Fedora COPR repository](https://copr.fedorainfracloud.org/coprs/rany/yggdrasil/) available, which provides recent versions of Yggdrasil - *only for Fedora!*
```

Comment:
Is *only for Fedora!* actually accurate? The COPR repo does list CentOS builds too.